scripts/standby_node_health_check/standby_node_health_check.py

```diff
@@ -0,0 +1,401 @@
+#
+# Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
+
+import re
+import subprocess
+import argparse
+import sys
+
+
+UFM_HA_CLUSTER_COMMAND = "ufm_ha_cluster {}"
+SYSTEMCTL_IS_ACTIVE_COMMAND = "systemctl is-active {}.service"
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="Checking if standby node is ready.")
+
+    parser.add_argument(
+        "--fabric-interfaces",
+        nargs="+",
+        required=True,
+        help="Specify one or more fabric interfaces (at least one is required)",
```

Comment:
I think it will be good to specific that for multiple interfaces, they should be comma separated, you can add an example.

Reply:
It is not a must, it works with comma and with whitespace (I tested). I will add an example.